src/TimeSelect.jsx

```diff
@@ -0,0 +1,128 @@
+'use strict';
+
+var React = require('react');
+var Input = require('react-bootstrap').Input;
```

Comment:
`UIToolkit` does not export a `select` type, I was recommended to stick with react-bootstrap as we didn't have a good reason for building that.